Dockerfile

```
@@ -27,6 +27,7 @@ ARG REPO=ThePalaceProject/virtual-library-card
 
 # Install system
 RUN apt-get update -y && \
+    apt-get install -y libexpat1 && \
     apt-get install --no-install-recommends -y \
     curl mime-support && \
```

One minor suggestion, you can install `libexpat1` along with the other deps, without adding a new `install` line:

```suggestion
    apt-get install --no-install-recommends -y \
    curl mime-support libexpat1 && \
```